### PR TITLE
Change shortcut window list order

### PR DESCRIPTION
**Pull request by AaronVanGeffen**
This changes the order of shortcuts displayed in the key binding window. Accomplished as follows:

* The list order in the shortcut window no longer depends on the internal implementation order.
* Added functionality for adding separators to the new list.
* Reordered shortcuts per @Umdlye's suggestions in #11223.

To close off, a few screenshots of the improved window in action. (It has been resized for screenshot purposes; the initial dimensions have not been changed.)

Over The Edge 2020-04-05 11-57-43
Over The Edge 2020-04-05 11-57-49
Over The Edge 2020-04-05 11-57-59

### CODE REVIEW

**Comment by IntelOrca** [comment]
I don't think these have to be done as part of this PR, since this doesn't change the file format. But I was thinking:

For the plugin system, I would like to allow custom shortcuts to be registered. For this I was thinking of making the shortcut IDs strings instead and storing them in a new text based file format (json or yaml) where we can preserve all the shortcuts even if you haven't loaded the plugin that registered it.

Additional to that, I would like to allow multiple bindings per shortcut and for mouse buttons to be possible bindings so that you can bind mouse 3 and 4 (usually the navigational buttons on the mouse).

**Comment by AaronVanGeffen** [comment]
These are great ideas. Could you also post them to the discussion in #11223? As you said, these are a little out of scope for this PR.

File: src/openrct2-ui/windows/ShortcutKeys.cpp [hunk 1]
```diff
@@ -283,6 +328,7 @@ static void window_shortcut_scrollmouseover(rct_window* w, int32_t scrollIndex,
 static void window_shortcut_scrollpaint(rct_window* w, rct_drawpixelinfo* dpi, int32_t scrollIndex)
 {
     gfx_fill_rect(dpi, dpi->x, dpi->y, dpi->x + dpi->width - 1, dpi->y + dpi->height - 1, ColourMapA[w->colours[1]].mid_light);
+    const auto scrollWidth = w->width - SCROLLBAR_WIDTH - 10;
```
Comment (AaronVanGeffen) [inline src/openrct2-ui/windows/ShortcutKeys.cpp:331]: Workaround for #11238

Comment (Gymnasiast) [reply src/openrct2-ui/windows/ShortcutKeys.cpp:331]: Can you add that as a comment?

Comment (AaronVanGeffen) [reply src/openrct2-ui/windows/ShortcutKeys.cpp:334]: Done.

**Comment by tupaschoal** [comment]
Needs rebase

**Comment by WackedSwe** [comment]
Somewhere along the way the copy /paste shortcuts in the list have been switched. Copy is not Paste and vice versa

**Pull request merged**